package.json

```diff
@@ -10,11 +10,11 @@
     "prepublish": "npm run build",
     "lint": "eslint --ext .js,.jsx src test stories",
     "storybook": "start-storybook -p 9001",
-    "pretest": "npm run lint",
-    "test": "npm run build && npm run test:coverage",
-    "tests-only": "npm run build && npm run test:quick",
-    "test:coverage": "babel-node node_modules/.bin/babel-istanbul cover node_modules/.bin/_mocha -- -R tap 'test/**/*-test.*'",
-    "test:quick": "babel-node node_modules/.bin/_mocha -R tap test/*-test.*",
+    "pretest": "npm run build && npm run lint",
+    "test": "nyc -e .jsx --reporter=lcov --reporter=text npm run test:unit",
+    "tests-only": "npm run build && npm run test:unit",
+    "test:unit": "SKIP_DOM=1 npm run test:_unit && npm run test:_unit -- --require './test/.dom.js'",
+    "test:_unit": "mocha -R tap 'test/**/*-test.*' --compilers js:babel-register",
```

Comment:
They're all unit tests. In reality a better name would be `mocha`, because you'd do `npm run mocha path/to/files` - let's go with that.